core/variant/typed_array.h

```diff
@@ -46,10 +46,15 @@ class TypedArray : public Array {
 		_ref(p_array);
 	}
 	_FORCE_INLINE_ TypedArray(const Variant &p_variant) :
-			Array(Array(p_variant), Variant::OBJECT, T::get_class_static(), Variant()) {
+			TypedArray(Array(p_variant)) {
```

Comment:
Simplified this by delegating to avoid copying code

Reply:
I'm all in to simplify the code.